python/psautohint/__init__.py

```diff
@@ -3,15 +3,28 @@
 import os
 import sys
 import plistlib
+import logging
 
 from fontTools.misc.py23 import tobytes, tounicode
 
 from . import _psautohint
 
 
+log = logging.getLogger(__name__)
+
 __version__ = _psautohint.version
 
 
+AUTOHINTEXE = os.path.join(
+    os.path.dirname(__file__),
+    "autohintexe" + (".exe" if sys.platform in ("win32", "cygwin") else "")
+)
+if not os.path.isfile(AUTOHINTEXE) or not os.access(AUTOHINTEXE, os.X_OK):
+    log.warning(
```

Comment:
it's better if you use `warnings.warn` here, instead of logging, as the latter might not be configured by the client application, and this warning is really directed not to the end-user of the tool, but to the developer integrating the psautohint library into an application, or to the system admin deploying and installing the package.
`warnings` is more about issues with the code itself, whereas logging more about what the code does

Reply:
Done, thanks!